kbpagesconfig/editor.go

```diff
@@ -0,0 +1,206 @@
+// Copyright 2017 Keybase Inc. All rights reserved.
+// Use of this source code is governed by a BSD
+// license that can be found in the LICENSE file.
+
+package main
+
+import (
+	"bytes"
+	"fmt"
+	"io"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/keybase/client/go/minterm"
+	"github.com/keybase/kbfs/libpages/config"
+	"github.com/sergi/go-diff/diffmatchpatch"
+	"golang.org/x/crypto/bcrypt"
+)
+
+const configFileName = ".kbp_config"
```

Comment:
Can we export the duplicate value from `libpages` and use it here?

Reply:
This package (intentionally) doesn't import `libpages`. But I guess we could move that to `config` package? The config itself doesn't care about what the file is called though. Let me know!

Reply:
Sure, `config` package seems reasonable even if it's not used there.